package/harvester-os/files/usr/share/rancher/rancherd/config.yaml.d/50-defaults.yaml

```diff
@@ -1,7 +1,7 @@
 rancherValues:
   rancherImagePullPolicy: IfNotPresent
   rancherImage: rancher/rancher
-  rancherImageTag: v2.6.0
+  rancherImageTag: v2.6.1-rc3
```

Comment:
Is RKE2 provisioning working in rc3? I don't think https://github.com/rancher/rancher/pull/34867 is included in rc3

Reply:
seems not, we need to bump Rancher to `v2.6-head` tagged version and fleet to the `0.3.7-rc1` respectively